Frends.LDAP.SearchObjects/Frends.LDAP.SearchObjects/SearchObjects.cs

```diff
@@ -86,6 +110,8 @@ public static Result SearchObjects([PropertyTab] Input input, [PropertyTab] Conn
         }
         catch (LdapException ex)
         {
+            if (connection.ThrowExceptionOnError)
+                throw;
             return new Result(false, ex.Message, null);
```

Comment:
In the error message, we should also include e.g. at least the exception class name (message does not necessarily have it embedded), so it could read "LdapException: {ex.Message}" or something similar